toolkit/data-sources/cli/Cargo.toml

```diff
@@ -17,7 +17,12 @@ workspace = true
 clap = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
-partner-chains-db-sync-data-sources = { workspace = true, features = ["block-source", "candidate-source"] }
+partner-chains-db-sync-data-sources = { workspace = true, features = [
+    "block-source",
+    "candidate-source",
+] }
+sidechain-mc-hash = { workspace = true }
+pallet-sidechain-rpc = { workspace = true }
```

Comment:
probably I should not include it directly and go through features of `partner-chains-db-sync-data-sources`?